op/main.py

```diff
@@ -11,6 +11,7 @@
 import op.model
 
 CHARM_STATE_FILE = '.unit-state.db'
+CHARM_CODE_FILE = '../lib/charm.py'
```

Comment:
That doesn't look right. Per agreements in the sprint, the charm.py file will be the entry point, and op/main.py inside a library. We cannot assume a relative path across them.

Reply:
In order to address this, I suggest this https://github.com/canonical/operator/pull/63/commits/5e99602d5281bec9b115fe765047755da8a28574

Please let me know if that looks right to you and, if so, I will add more test coverage.